code/game/objects/items/binoculars.dm

```diff
@@ -214,13 +214,8 @@
 		return
 	var/is_outside = FALSE
 	if(is_ground_level(TU.z))
-		switch(targ_area.ceiling)
-			if(CEILING_NONE)
-				is_outside = TRUE
-			if(CEILING_GLASS)
-				is_outside = TRUE
-			if(CEILING_METAL)
-				is_outside = TRUE
+		if (targ_area.ceiling <= CEILING_OBSTRUCTED)
+			is_outside = TRUE
```

Comment:
Just combine this into a single line. if ground side and ceiling less than obstructed

Reply:
> Just combine this into a single line. if ground side and ceiling less than obstructed

no because im pretty sure shipside maps dont have ceilings and if i ask both at once i think itll be mad and runtime or something which is why i did it this way

Reply:
bruh.

Reply:
> bruh, its false by default.

i will trust you